### ISSUE TITLE
draft-js-import-markdown doesn't display images

### ISSUE THREAD

**Issue by kildem** [sstur/draft-js-utils#147]
I have try this test: https://github.com/sstur/draft-js-utils/commit/503e01220681c40892191bc53cf55d2ba771e94a#diff-e009bb42daadca1e8c6103199d3b13b1

It says that it should support images. But this obj:
```
{
  blocks: [{
         data: {},
         depth: 0,
         entityRanges: [{
           key: 0,
           length: 1,
           offset: 0,
         }],
         inlineStyleRanges: [],
         text: ' ',
         type: 'unstyled',
       }],
       entityMap: {
         // This is necessary due to flow not supporting non-string literal property keys
         // eslint-disable-next-line quote-props
         '0': {
           data: {
             src: 'imgur.com/asdf.jpg',
           },
           mutability: 'MUTABLE',
           type: 'IMAGE',
         }
     }
}
```
doesn't show images.
It is missing atomic block in order to display image.
this one display image:
```
      const obj = {
 blocks: [
          {
            text: ' ',
            type: 'atomic',
            depth: 0,
            inlineStyleRanges: [],
            entityRanges: [
              {
                offset: 0,
                length: 1,
                key: 0
              }
            ],
            data: {},
            key: 'foo'
          },
          {
            text: '',
            type: 'unstyled',
            depth: 0,
            inlineStyleRanges: [],
            entityRanges: [],
            data: {},
            key: 'foo1'
          }
        ],
        entityMap: {
          // This is necessary due to flow not supporting non-string literal property keys
          // eslint-disable-next-line quote-props
          '0': {
            type: 'IMAGE',
            mutability: 'MUTABLE',
            data: {
              src: 'imgur.com/asdf.jpg',
            }
          }
        }
      }
      //return EditorState.createWithContent(convertFromRaw(obj))
     ```

**Comment by charlimxm** [comment]
bump! same issue for me

**Comment by lifeisfoo** [comment]
As you can see from [draft-js-image-plugin source](https://github.com/draft-js-plugins/draft-js-plugins/blob/master/draft-js-image-plugin/src/index.js), the [image plugin](https://www.draft-js-plugins.com/plugin/image) need an `atomic` block of type `IMAGE` to render the image in the editor.

**Comment by kildem** [comment]
it works if I replace 'unstyled' with 'atomic'

**Comment by mxstbr** [comment]
I am having the same issue! @sstur any ideas how we could make images render as atomic blocks?

**Comment by mxstbr** [comment]
Fixed in #164 by generating `<figure>` for images from markdown!

**Comment by Harpreetdhaner** [comment]
https://play-lh.googleusercontent.com/1-hPxafOxdYpYZEOKzNIkSP43HXCNftVJVttoo4ucl7rsMASXW3Xr6GlXURCubE1tA=w3840-h2160-rw
